src/main/kotlin/no/nav/tilleggsstonader/klage/fullmakt/FullmaktService.kt

```diff
@@ -0,0 +1,34 @@
+package no.nav.tilleggsstonader.klage.fullmakt
+
+import no.nav.tilleggsstonader.klage.felles.util.isEqualOrBefore
+import no.nav.tilleggsstonader.kontrakter.felles.Tema
+import no.nav.tilleggsstonader.kontrakter.fullmakt.FullmektigDto
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Component
+import java.time.LocalDate.now
+
+@Component
+class FullmaktService(
+    val fullmaktClient: FullmaktClient,
```

Comment:
Disse kan være private for å ikke være tilgjengelige fra konsumenter av servicen
```suggestion
    private val fullmaktClient: FullmaktClient,
```